### PR TITLE
[OSDOCS-7284.1]: Numbered list in documentation incorrectly formatted

### PR DESCRIPTION
**Pull request by mletalie**
Version(s):

4.12+
Issue:

https://issues.redhat.com/browse/OSDOCS-7284
Link to docs preview:

https://63106--docspreview.netlify.app/openshift-enterprise/latest/machine_management/applying-autoscaling#cluster-autoscaler-cr_applying-autoscaling
QE review:
- [ ] QE has approved this change. QE approval N/A

Additional information:

image

### CODE REVIEW

File: modules/cluster-autoscaler-cr.adoc [hunk 1]
```diff
@@ -65,7 +65,8 @@ If you do not specify a value, the default value of `1` is used.
...
 <14> Optional: Specify the period to wait before deleting a node after a node has recently been _deleted_. If you do not specify a value, the default value of `0s` is used.
 <15> Optional: Specify the period to wait before deleting a node after a scale down failure occurred. If you do not specify a value, the default value of `3m` is used.
-<16> Optional: Specify the period before an unnecessary node is eligible for deletion. If you do not specify a value, the default value of `10m` is used.<17> Optional: Specify the _node utilization level_ below which an unnecessary node is eligible for deletion. The node utilization level is the sum of the requested resources divided by the allocated resources for the node, and must be a value greater than `"0"` but less than `"1"`. If you do not specify a value, the cluster autoscaler uses a default value of `"0.5"`, which corresponds to 50% utilization. This value must be expressed as a string.
+<16> Optional: Specify the period before an unnecessary node is eligible for deletion. If you do not specify a value, the default value of `10m` is used.
```
Comment (sabrinajess) [inline modules/cluster-autoscaler-cr.adoc:68]: Edit for clarity: update `period` to:

- a specific unit of time
- `period of time`
- `time period` 
- or similar.

File: modules/cluster-autoscaler-cr.adoc [hunk 2]
```diff
@@ -65,7 +65,8 @@ If you do not specify a value, the default value of `1` is used.
...
+<17> Optional: Specify the _node utilization level_ below which an unnecessary node is eligible for deletion. The node utilization level is the sum of the requested resources divided by the allocated resources for the node, and must be a value greater than `"0"` but less than `"1"`. If you do not specify a value, the cluster autoscaler uses a default value of `"0.5"`, which corresponds to 50% utilization. This value must be expressed as a string.
```
Comment (sabrinajess) [inline modules/cluster-autoscaler-cr.adoc:69]: I found the first sentence difficult to read. Optional suggestion: 

<17> Optional: Specify the _node utilization level_. Nodes below this utilization level are eligible for deletion.

**Review by sabrinajess — COMMENTED** [review]
Some clarity nits, otherwise LGTM. Tagging @ousleyp in for any additional feedback.

**Comment by mletalie** [comment]
Hello @sabrinajess,
   Thank you taking a look at this; I did update the doc based on your suggestions. ML

**Comment by ousleyp** [comment]
Nice work; LGTM with the changes! Great suggestions, @sabrinajess :)

**Comment by mletalie** [comment]
Thanks @ousleyp and @sabrinajess!

**Pull request merged**